edx_reverification_block/xblock/reverification_block.py

```diff
@@ -65,6 +68,13 @@ class ReverificationBlock(XBlock):
         help="ISO-8601 formatted string representing the due date of this related assessment."
     )
 
+    grace_period = Integer(
+        display_name="Verification Grace Period",
+        scope=Scope.settings,
+        default=ICRV_GRACE_PERIOD,
+        help="The number of days learners can verify their identity after the exam's due date."
```

Comment:
I believe it is, although I don't know for sure. If I remember correctly, there are still some parts of XBlock that haven't gotten the love and attention they deserve, and one of those parts is around internationalization and string formatting of help strings.